dask_kubernetes/tests/test_core.py

```diff
@@ -29,59 +29,61 @@
 
 
 @pytest.fixture
-def api():
-    ClusterAuth.load_first()
-    return kubernetes.client.CoreV1Api()
+async def api():
+    await ClusterAuth.load_first()
+    yield kubernetes.client.CoreV1Api()
 
 
 @pytest.fixture
-def ns(api):
-    name = "test-dask-kubernetes" + str(uuid.uuid4())[:10]
+async def ns(api):
+    name = "test-dask-kubernetes-" + str(uuid.uuid4())[:4]
     ns = kubernetes.client.V1Namespace(
         metadata=kubernetes.client.V1ObjectMeta(name=name)
     )
-    api.create_namespace(ns)
+    await api.create_namespace(ns)
     try:
         yield name
     finally:
-        api.delete_namespace(name)
+        await api.delete_namespace(name)
 
 
 @pytest.fixture
-def pod_spec(image_name):
+async def pod_spec(image_name):
     yield make_pod_spec(
         image=image_name, extra_container_config={"imagePullPolicy": "IfNotPresent"}
     )
 
 
 @pytest.fixture
-def clean_pod_spec(pod_spec):
+async def clean_pod_spec(pod_spec):
```

Comment:
Given that there are no `await` statements in this function you can probably drop the `async` as well.